templates/frontend/objects/article_summary.tpl

```diff
@@ -21,13 +21,13 @@
 	{assign var="showAuthor" value=true}
 {/if}
 
-<div class="article_summary">
+<article class="article_summary">
 	<div class="article_summary_body">
-		<div class="summary_title_wrapper">
+		<h5 class="summary_title_wrapper">
```

Comment:
Are you sure that the `h5` will be correct every place this template is used? I believe it's used on the homepage (when showing the current issue), on the issue TOC page, and in the search results. I have often left the `h*` out to ensure the wrong heading isn't used somewhere by accident. But another approach you can use is to pass the heading into the template when it is used. For example, in the issue TOC, something like:
```
{include file="frontend/objects/article_summary.tpl" headingLevel="3"}
```
Then something like:
```
{if $headingLevel}
  <h{$headingLevel} class="summary_title_wrapper">
{else}
  <div class="summary_title_wrapper">
{/if}
```